v2/models/addressable.go

```diff
@@ -0,0 +1,41 @@
+//
```

Comment:
Per Core WG discussion, this can be removed.

Reply:
Removed.